static/app/gettingStartedDocs/node/nestjs.tsx

```diff
@@ -0,0 +1,182 @@
+import ExternalLink from 'sentry/components/links/externalLink';
+import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import type {
+  Docs,
+  DocsParams,
+  OnboardingConfig,
+} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {getUploadSourceMapsStep} from 'sentry/components/onboarding/gettingStartedDoc/utils';
+import {
+  getCrashReportApiIntroduction,
+  getCrashReportInstallDescription,
+  getCrashReportJavaScriptInstallStep,
+  getCrashReportModalConfigDescription,
+  getCrashReportModalIntroduction,
+} from 'sentry/components/onboarding/gettingStartedDoc/utils/feedbackOnboarding';
+import {getJSServerMetricsOnboarding} from 'sentry/components/onboarding/gettingStartedDoc/utils/metricsOnboarding';
+import {t, tct} from 'sentry/locale';
+import {getInstallConfig, getSdkInitSnippet} from 'sentry/utils/gettingStartedDocs/node';
+
+type Params = DocsParams;
+
+const getSdkSetupSnippet = () => `
+// IMPORTANT: Make sure to import \`instrument.js\` at the top of your file.
+import './instrument';
+
+import * as Sentry from '@sentry/node';
```

Comment:
```suggestion

// All other imports below
import * as Sentry from '@sentry/node';
```

let's add a comment like this, this ensures this is not reordered e.g. by vscode!